docs/user/guides/index.rst

```diff
@@ -1,16 +1,12 @@
 :orphan:
 
-Guides
-======
+How-to guides A-Z
```

Comment:
Not sure what A-Z means? What is this page doing, do we need it if it isn't including the TOCTrees?

Reply:
I think that `A-Z` is a good way to say "here is just a full listing".

There's a separate task for this, though. I found it too hard to jump into without a good structure for meta data - titles and intro paragraphs.

> Is an index page possible or useful at this stage? Move to separate task

Reply:
Leaving this for now, it was an orphan before, it still is.